testdata/p4_16_samples_outputs/pna-example-addhit.p4.spec

```diff
@@ -0,0 +1,194 @@
+
```

Comment:
This file shouldn't have been generated for error cases.

Reply:
I don't know how to fix this - any idea?